source/tyr/tests/integration/conftest.py

```diff
@@ -169,12 +169,14 @@ def create_repositories_instance_env_variables():
     backup_directory = "{path}/ed/backup".format(path=tmp_path)
     aliases_directory = "{path}/ed/aliases".format(path=tmp_path)
     synonyms_directory = "{path}/ed/synonyms_file".format(path=tmp_path)
```

Comment:
Shouldn't you remove those from tests as much as possible, to check it's safe?

Reply:
Done